sb_wysiwyg.features.ckeditor_profile.inc

```diff
@@ -83,7 +167,7 @@ function sb_wysiwyg_ckeditor_profile_defaults() {
         'html_entities' => 'f',
         'scayt_autoStartup' => 'f',
         'theme_config_js' => 't',
-        'js_conf' => '',
+        'js_conf' => 'config.image2_alignClasses = [ \'media-left\', \'center-block\', \'media-right\' ];',
```

Comment:
@jeffam I'm not entirely sure that we want to use media-left and media-right. That is part of the bootstrap [media-component](http://getbootstrap.com/components/#media) In a context outside of the media component, it might make more sense to use pull-right and pull-left. Might want to confirm with Gennie